tests/Sinch.Tests/Core/HttpTests.cs

```diff
@@ -129,5 +131,34 @@ public async Task NewTokenFetchedIfWwwExpiredIsPresent()
             await response.Should().NotThrowAsync();
             _httpMessageHandlerMock.VerifyNoOutstandingExpectation();
         }
+
+        [Fact]
+        public async Task SendSinchHeader()
+        {
+            _tokenManagerMock
+                .GetAuthToken(Arg.Any<bool>())
+                .Returns("first_token");
+
+            var uri = new Uri("http://sinch.com/items");
+
+            var sdkVersion = new AssemblyName(typeof(Http).GetTypeInfo().Assembly.FullName).Version.ToString();
+
+            _httpMessageHandlerMock.Expect(HttpMethod.Get, uri.ToString())
+                .WithHeaders("Authorization", "Bearer first_token")
+                // somehow the matchers split header on whitespaces and check it by parts
+                // so,  .WithHeaders("User-Agent", $"sinch-sdk/{sdkVersion} (csharp/{RuntimeInformation.FrameworkDescription};;)")
+                // not working
```

Comment:
Because of not working from `_httpMessageHandlerMock.Expect`, may be a unit test around the generated string from https://github.com/sinch/sinch-sdk-dotnet/pull/29/files#diff-db7c40e4ea30fad828578dddf4cfe82d12b2185db1770e4fc4d24a416fadaf96R71
Using a dedicated function and unit test checking against expected value/format ?

Reply:
It's working, , just in a weird way splitting the content on whitespace when checking, so in fact it checks two parts of User-Agent to match a request. I deem it to be fine to check

Reply:
Does this mean both:
- `sinch-sdk/{sdk_version} ({language}/{language_version}; {implementation_type}; {auxiliary_flag})`
- `{language}/{language_version}; {implementation_type}; {auxiliary_flag}) sinch-sdk/{sdk_version}`

will pass the test but content for second one is wrong (not compliant with expected format) ?

Reply:
@JPPortier added matcher for exact sequence of header values for the case

Reply:
Great thank you